No owner for specific file

if i create a codeowners file like this:
CODEOWNERS:
/build/logs/log.txt @ghost

owners = CodeOwners(example_file)
owners.of("/build/logs/log.txt") it doesnt return any owner

was this supposed to happen?

Maybe this is another case of behavior differing from git?

Git ignore says it doesn't match:

```
_______________________________________________________________ test_specific_patterns_against_git[single-file regression] _______________________________________________________________

name = 'single-file regression', pattern = '/build/logs/log.txt', paths = {'/build/logs/log.txt': True}

    @pytest.mark.parametrize(
        "name,pattern,paths", GO_CODEOWNER_EXAMPLES, ids=ids_for(GO_CODEOWNER_EXAMPLES)
    )
    def test_specific_patterns_against_git(
        name: str, pattern: str, paths: Dict[str, bool]
    ) -> None:
        """
        Ensure the expected patterns match actual git behavior.
    
        Codeowners is a subset of git ignore behavior so checking against it
        should work in most cases.
        """
        if name == "docs with star":
            pytest.skip("Behaviour for docs-with-star does not match gitignore")
    
        assert paths
        with tempfile.TemporaryDirectory() as directory:
            subprocess.run(["git", "init"], cwd=directory, check=True, capture_output=True)
            (Path(directory) / ".gitignore").write_text(pattern + "\n")
            for path, expected in paths.items():
                res = subprocess.run(
                    ["git", "check-ignore", path], cwd=directory, capture_output=True
                )
                actual = res.returncode == 0
>               assert (
                    actual is expected
                ), f"match for pattern:{pattern} and path:{path} failed, expected: {expected}, actual: {actual}"
E               AssertionError: match for pattern:/build/logs/log.txt and path:/build/logs/log.txt failed, expected: True, actual: False
E               assert False is True
```

--- a/README.md
+++ b/README.md
@@ -37,11 +37,8 @@ assert owners.of("test.js") ==  [('USERNAME', '@ghost')]
 
 ### Path format
 
-Paths passed to `of()` (and `matching_line()` / `section_name()`) must be
-repo-relative with no leading slash, matching the behavior of the upstream
-[Go](https://github.com/hmarr/codeowners/) and
-[Rust](https://github.com/hmarr/codeowners-rs) libraries as well as
-`git check-ignore`. A leading slash will not match:
+Paths must be repo-relative with no leading slash, matching `git check-ignore`. A leading
+slash will not match:
 
 ```python
 owners = CodeOwners("/build/logs/log.txt @ghost")

--- a/README.md
+++ b/README.md
@@ -35,6 +35,20 @@ owners = CodeOwners(example_file)
 assert owners.of("test.js") ==  [('USERNAME', '@ghost')]
 ```
 
+### Path format
+
+Paths passed to `of()` (and `matching_line()` / `section_name()`) must be
+repo-relative with no leading slash, matching the behavior of the upstream
+[Go](https://github.com/hmarr/codeowners/) and
+[Rust](https://github.com/hmarr/codeowners-rs) libraries as well as
+`git check-ignore`. A leading slash will not match:
+
+```python
+owners = CodeOwners("/build/logs/log.txt @ghost")
+assert owners.of("build/logs/log.txt") == [('USERNAME', '@ghost')]  # ✅
+assert owners.of("/build/logs/log.txt") == []                       # ✗ leading slash
+```
+
 ## Dev
 
 ```shell

